dotnet/tests/Microsoft.Agents.M365Copilot.Core.Tests/TestModels/ServiceModels/TestAttendee.cs

```diff
@@ -30,7 +30,7 @@ public TestAttendee()
         }
 
         /// <summary>
-        /// Serialize the <see cref="TestChangeNotificationEncryptedContent"/> instance
+        /// 
```

Comment:
Either restore the doc comment or remove the <summary>

Reply:
Fix in #84